ui/src/components/settings/SettingTable.js

```diff
@@ -355,6 +366,30 @@ class SettingTable extends React.Component {
                 />
             );
 
+        let auditEnabledDesc =
+            'Flag indicates whether or not ' +
+            this.props.category +
+            ' updates require explicit auditing approval process';
+        this.props.category === 'role' &&
+            this.props.justificationRequired &&
+            rows.push(
+                <StyledSettingRow
+                    key={'setting-row-auditEnabled'}
+                    disabled={
+                        this.state.originalCollectionDetails.auditEnabled ||
+                        this.state.copyCollectionDetails.hasRoleMembers
```

Comment:
need to add this.state.copyCollectionDetails.hasGroupMembers

Reply:
fixed! apparently i didn't push the changes from vzm-ui, but now its synched.